source/faq/faq.rst

```diff
@@ -123,115 +119,139 @@ Devices
 What are *devices* for dojot?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In dojot, a device is a digital representation of an actual device or gateway with one 
-or more sensors or of a virtual one with sensors/attributes inferred from other devices.
+In dojot, a device is a digital representation of an actual device or gateway
+with one or more sensors or of a virtual one with sensors/attributes inferred
+from other devices.
 
-Consider, for instance, an actual device with temperature and humidity sensors; it can be 
-represented into dojot as a device with two attributes (one for each sensor). We call this 
-kind of device as *regular device* or by its communication protocol, for instance, 
-*MQTT device* or *CoAP device*.
+Consider, for instance, an actual device with temperature and humidity sensors;
+it can be represented into dojot as a device with two attributes (one for each
+sensor). We call this kind of device as *regular device* or by its
+communication protocol, for instance, *MQTT device* or *CoAP device*.
 
-We can also create devices which don’t directly correspond to their actual ones, for instance, 
-we can create one with higher level of information of temperature (*is becoming hotter* or 
-*is becoming colder*) whose values are inferred from temperature sensors of other devices. 
-This kind of device is called *virtual device*.
+We can also create devices which don’t directly correspond to their actual
+ones, for instance, we can create one with higher level of information of
```

Comment:
> actual ones
physical counterparts / real versions
>  we can create one with higher level of information of temperature
we can create one with a higher level of temperature information (is getting hotter or ...)